Change the compile-time MAXVIFS constant for more mcast interfaces

diff --git a/include/linux/mroute.h b/include/linux/mroute.h
index 46caaf4..3287cb1 100644
--- a/include/linux/mroute.h
+++ b/include/linux/mroute.h
@@ -34,7 +34,7 @@
 #define SIOCGETSGCNT	(SIOCPROTOPRIVATE+1)
 #define SIOCGETRPF	(SIOCPROTOPRIVATE+2)
 
-#define MAXVIFS		32	
+#define MAXVIFS		256
 typedef unsigned long vifbitmap_t;	/* User mode code depends on this lot */
 typedef unsigned short vifi_t;
 #define ALL_VIFS	((vifi_t)(-1))
